src/main/resources/data/contenttweaker/scripts/contenttweaker_advanced_function.zs

```diff
@@ -0,0 +1,22 @@
+#loader crafttweaker
+// Advanced functions is called on game runtime.
+// So set function actions should be run in loader crafttweaker and reloadable.
+// Since the default loader is crafttweaker, you can omit the loader preprocessor, just clarify it here.
+
+// Uses BEP to get an IIsCotItem and IIsCotBlock instance.
+// <cotitem:generic_item>
+// <cotblock:generic_block>
+
+// Makes the item like a infinite duration flint and steel
+// The on item use function is called whenever the item is used on a block.
+// For more info, see documentation.
+<cotitem:generic_item>.setOnItemUse((context) => {
```

Comment:
If you want to make another flint and steel, why not give the item a matching resource name then 😉